### PR TITLE
Minor README.md updates

### PR DESCRIPTION
**Pull request by ccostino**
This changeset makes a few more adjustments to the README.md file to account for updates in the other repos that are forthcoming.

## Security Considerations

- None with this change.

### CODE REVIEW

**Review by terrazoon — APPROVED** [review]
looks good to me

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# US Notify API
+# US Notify.gov API
```
Comment (heyitsmebev) [inline README.md:1]: Isn't it now "Notify.gov" and not "US Notify.gov"

**Review by A-Shumway42 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (ccostino) [reply README.md:1]: Yes, thanks for flagging this!

**Pull request merged**